docs/persistent-volumes.md

```diff
@@ -0,0 +1,170 @@
+# Persistent Volumes and Claims
+
+This document describes the current state of Persistent Volumes in Kubernetes.  Familiarity with [volumes](./volumes.md) is suggested.
+
```

Comment:
I would segue into this with some background like 

Managing storage is a distinct problem from managing compute.  The `PersistentVolume` subsystem provides an API for users and administrators to abstract the details of how storage is provided from how it is consumed.  To do this we introduce two new API resources: